content/dev/dev-guide.md

```diff
@@ -0,0 +1,264 @@
+# Tlon Dev Guide
+## What is this guide?
+This is Tlon's guide for new developers that want to build applications on Urbit. The goal of this guide is to take you from a fresh machine to a working development environment for Urbit applications. This guide assumes macOS, but Linux is also supported (see the Urbit foundation docs for Linux support) It's important this guide remains clear and correct, if you find something broken please let us know at support@tlon.io
+
+## Pre-requisites
+
+### Homebrew
+- Homebrew is a package manager for macOS, we'll need it installed in order to install `git-lfs`.
+- ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"```
+
+### Git LFS
+- A package that enables git to work with large files, we'll install this via brew.
+- ```brew install git-lfs```
+
+### Wget
+ - A package used to download artifacts from the web
+ - `brew install wget`
+
+### Visual Studio Code
+ - While not technically a prerequisite, there's a hoon syntax highlighting plugin available and this is the editor we recommend: https://code.visualstudio.com/
+ - For the small text file edits in this guide, we'll simply use `nano`.
+
+### Urbit Install
+ - First we need to create a directory for the urbit binary, we'll create this in your home directory.
+   - ```bash
+     mkdir ~/urbit-dev
+     ```
+ - Next we're going to pull down the latest urbit binary from urbit.org from the newly created directory.
+    - ```bash
+      cd ~/urbit-dev
+      curl -JLO https://urbit.org/install/mac/latest
+      ```
+  - Now we'll untar the contents of the download into our newly created directory.
+    - ```bash 
+      tar zxvf ./darwin.tgz --strip=1
```

Comment:
```suggestion
      tar -zxvf ./darwin.tgz --strip=1
```

Reply:
I think you don't need the `-` for tar, but I've added it anyway since it works either way.

Reply:
TIL, cool